starlink/templates/main/index.html

```diff
@@ -23,7 +23,7 @@
         {% for id, details in listDict.items() %}
         <tr>
           <td>{{ details['dateAdded'] }} <p hidden>{{ details['id'] }}</p></td>
-          <td>{{ details['version'] }}</td>
+          <td><tt><b>{{ details['version'][:8] }}</b>{{ details['version'][8:] }}</tt></td>
```

Comment:
<tt> tag seems to be depreciated in html5. 

Do we need this?

Reply:
I'd like it fixed width, but any way to set the font is fine. <code> in the Bootstrap CSS changes the color too which I didn't want.